streamingccl: lint failure on deprecation warning

lint_test.go:1751:
        pkg/ccl/streamingccl/streamclient/partitioned_stream_client_test.go:14:2: "io/ioutil" has been deprecated since Go 1.16: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details.  (SA1019)
    --- FAIL: TestLint/TestStaticCheck (974.77s)


Jira issue: CRDB-21955

--- a/pkg/ccl/streamingccl/streamclient/partitioned_stream_client_test.go
+++ b/pkg/ccl/streamingccl/streamclient/partitioned_stream_client_test.go
@@ -11,8 +11,8 @@ package streamclient
 import (
 	"context"
 	"fmt"
-	"io/ioutil"
 	"net/url"
+	"os"
 	"strings"
 	"testing"
 	"time"
@@ -107,7 +107,7 @@ INSERT INTO d.t2 VALUES (2);
 		v := ret.Query()
 		for _, opt := range []string{"sslcert", "sslkey", "sslrootcert"} {
 			path := v.Get(opt)
-			content, err := ioutil.ReadFile(path)
+			content, err := os.ReadFile(path)
 			require.NoError(t, err)
 			v.Set(opt, string(content))
 

